debian/bitlbee-common.config

```diff
@@ -1,21 +0,0 @@
-#!/bin/sh
-
-set -e
-
-. /usr/share/debconf/confmodule
-[ -f /etc/default/bitlbee ] && . /etc/default/bitlbee
-
-db_title BitlBee
-
-if [ -n "$BITLBEE_PORT" ]; then
```

Comment:
Right. I was going to do that as suggested by #debian-devel, and half-forgot about it, half-thought that leaving a note in /etc/default/bitlbee itself is enough.

I actually like the idea of automatically migrating it, will give that a shot.